react/OrderItems.tsx

```diff
@@ -39,7 +43,8 @@ interface EnqueuedTask {
 
 const OrderItemsContext = createContext<Context | undefined>(undefined)
 
-const noop = async (_: Partial<Item>) => {}
+const noopItem = async (_: Partial<Item>) => {}
+const noopItems = async (_: Partial<Item>[]) => {}
```

Comment:
How about `const noop = async (_: any) => {}`? I think this way you don't need to create two functions.